src/Handler/CanNotInvokeHandler.php

```diff
@@ -39,12 +35,15 @@ public static function forCommand($command, string $reason)
         return $exception;
     }
 
+    private function __construct(string $message = '', $code = 0, Throwable $previous = null)
+    {
+        parent::__construct($message, $code, $previous);
+    }
+
     /**
      * Returns the command that could not be invoked
-     *
-     * @return mixed
      */
-    public function getCommand()
+    public function getCommand(): object
     {
         return $this->command;
     }
```

Comment:
Other than that one thing, I love everything you've done in this file! :heart: :rainbow: :unicorn: :star2: